### PR TITLE
feat(sms-limiting): changes for mail/modal

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
1. Update copywriting for emails 
2. Update `reply-to` in email
3. Update copywriting for modal (not yet confirmed)
4. log email disablings/sends

### CODE REVIEW

File: src/app/views/templates/sms-verification-disabled-admin.server.view.html [hunk 1]
```diff
@@ -4,20 +4,21 @@
...
     <p>
-      As you have reached the free tier limit of <%= smsVerificationLimit %>
-      verifications sent, SMS OTP verification has been
-      <b>automatically disabled</b> on all your form(s). Respondents will still
-      be able to submit to your form(s) but will not be prompted to verify their
-      mobile numbers.
+      <b>
+        As you have reached the free tier limit of <%= smsVerificationLimit %>
+        per account, SMS OTP verification has been automatically disabled on all
+        forms you own.
```
Comment (karrui) [inline src/app/views/templates/sms-verification-disabled-admin.server.view.html:18]: I thought SMS OTP verification will only be disabled on forms that are NOT onboarded yet? Not all forms they own

File: src/app/views/templates/sms-verification-warning.view.html [hunk 2]
```diff
@@ -4,21 +4,23 @@
...
       <li><a href="<%= link %>"> <%= title %> </a></li>
       <% }) %>
     </ol>
     <p>
-      Your account can use <b><%= numAvailable %></b> more free verifications
-      until free SMS OTP verification is automatically disabled for all owned
-      forms.
+      <b>
+        Your account can use <%= numAvailable %> more free verifications until
+        free SMS OTP verification is automatically disabled for all owned forms.
```
Comment (karrui) [inline src/app/views/templates/sms-verification-warning.view.html:18]: same here

File: src/app/views/templates/sms-verification-disabled-admin.server.view.html (hunk 1 above)
Comment (seaerchin) [reply src/app/views/templates/sms-verification-disabled-admin.server.view.html:18]: thx 4 catch, fixed!

File: src/app/views/templates/sms-verification-warning.view.html [hunk 3]
```diff
@@ -17,6 +17,7 @@
         Your account can use <%= numAvailable %> more free verifications until
         free SMS OTP verification is automatically disabled for all owned forms.
```
Comment (karrui) [inline src/app/views/templates/sms-verification-warning.view.html:18]: I think we should just wait for sonjia for the copywriting lol! Here you put disabled for all owned forms. Then below you put Forms with Twilio not affected. Very confusing sentences.

Comment (seaerchin) [reply src/app/views/templates/sms-verification-warning.view.html:18]: this is her copy writing HAHAHA cos ytd i realized the same thing you did and asked sonjia to update the copy but i forgot to update after she updated. 

only the modal's copywriting isn't approved by sonjia

Comment (karrui) [reply src/app/views/templates/sms-verification-warning.view.html:18]: Please double check this! Reads awkwardly

**Review by karrui — APPROVED** [review]
lgtm

**Pull request merged**